app/src/main/java/com/github/leuludyha/ibdb/FirebaseActivity.kt

```diff
@@ -0,0 +1,98 @@
+package com.github.leuludyha.ibdb
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.github.leuludyha.ibdb.ui.theme.IBDBTheme
+import com.google.firebase.database.DatabaseReference
+import com.google.firebase.database.ktx.database
+import com.google.firebase.ktx.Firebase
+import java.util.concurrent.CompletableFuture
+
+private val db: DatabaseReference = Firebase.database.reference
```

Comment:
As it's for the bootcamp I don't think there is any problem putting it in this file only but was there any suggestion in the documentation/bootcamp instructions as how to access such an object we need in the whole application ?

Reply:
Good point, we can keep it here for now and then move it to another file later to be able to use it in other parts. There was nothing about this in the instructions